tests/test_translation_level_08.py

```diff
@@ -15,68 +15,72 @@ class TestsTranslationLevel8(HedyTester):
     keywords_from = hedy_translation.keywords_to_dict('en')
     keywords_to = hedy_translation.keywords_to_dict('nl')
 
-    @check_local_lang_bool
     def test_repeat_indent_english_dutch(self):
-        code = "repeat 3 times\n" \
-               "    print 'Hedy is fun!'"
+        code = textwrap.dedent("""\
+                                repeat 3 times
+                                    print 'Hedy is fun!'""")
 
         result = hedy_translation.translate_keywords(code, from_lang="en", to_lang="nl", level=self.level)
-        expected = "herhaal 3 keer\n" \
-                   "    print 'Hedy is fun!'"
+        expected = textwrap.dedent("""\
+                                    herhaal 3 keer
+                                        print 'Hedy is fun!'""")
 
         self.assertEqual(result, expected)
 
-    @check_local_lang_bool
     def test_repeat_multiple_indent_english_dutch(self):
-        code = "repeat 3 times\n" \
-               "    print 'Hedy is fun!'\n" \
-               "    print 'print 3 keer'\n" \
-               "print 'print 1 keer'"
+        code = textwrap.dedent("""\
+                                repeat 3 times
```

Comment:
this is ok, but I would align it with the assignment like this, that is a bit more readable:
```suggestion
        code = textwrap.dedent("""\
        repeat 3 times
        etc...
```